CollAction/Services/ProjectEmailOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace CollAction.Services
+{
+    public class ProjectEmailOptions
+    {
+        public int MaxNumberProjectEmails { get; set; }
```

Comment:
I think we need a default for this (probably equal to the maximum number of participants allowed on a project), or don't allow the application to start if it's not set. Now it defaults to 0, which means this function won't work. Something that could easily go wrong some time...

Reply:
Added some defaults